ci/docker/install/ubuntu_scala.sh

```diff
@@ -24,8 +24,9 @@ set -ex
 cd "$(dirname "$0")"
 # install libraries for mxnet's scala package on ubuntu
 echo 'Installing Scala...'
+apt-get update || true
 apt-get install -y software-properties-common
-apt-get update
+apt-get update || true
```

Comment:
Just double checking.  Is the reason we do two updates here that software-properities-common adds some new repos for apt?

Reply:
correct